Makefile

```diff
@@ -11,6 +11,12 @@ PKGS = $(shell go list ./... | grep -v /vendor/)
 gofmtcheck:
 	@go fmt $(PKGS) | grep ".*\.go"; if [ "$$?" = "0" ]; then exit 1; fi     
 
+doccheck:
+	grep -hoE '(kube_[^ |]+)' Documentation/* | sort -u > documented_metrics
```

Comment:
Let's drop a little comment in here so that we 3 months from now still know what this make target is doing :slightly_smiling_face:, maybe just something small like "This make target extracts the metrics from the documentation by taking all strings starting with `kube_` and compares it with the metrics it can find by taking all `kube_` strings it can find in the tests." I'm sure we can write this better, just an English description would be great for someone who gets a failing CI job.